Client/src/js/components/EditDialog.jsx

```diff
@@ -36,10 +36,10 @@ var EditDialog = React.createClass({
 
     return <ModalDialog backdrop="static" className={ `edit-dialog ${this.props.className || ''}` } { ...props } footer={
       <span>
-        <Button onClick={ this.props.onClose }>{ this.props.closeText || 'Close' }</Button>
         {
           this.props.readOnly || <Button bsStyle="primary" onClick={ this.save }>{ this.props.saveText || 'Save' }</Button>
         }
+        <Button onClick={ this.props.onClose }>{ this.props.closeText || 'Close' }</Button>
```

Comment:
The Close and Save button order has been reversed.  My bad on this direction.  We should keep the consistency from the previous version.

I'll create a separate pull request to resolve this.